src/wazuh_db/wdb_parser.c

```diff
@@ -5523,6 +5525,8 @@ int wdb_parse_global_delete_agent(wdb_t * wdb, char * input, char * output) {
     int agent_id = 0;
 
     agent_id = atoi(input);
+    char padded_agent_id[AGENT_ID_LEN];
```

Comment:
This will fail if the agent is greater.
We need to pad to at least 3 characters without affecting the current behavior.

Reply:
I use the same logic that above, setting a char[] of 64 lenght